projects/common/src/utilities/url/url-utilities.ts

```diff
@@ -0,0 +1,5 @@
+import { HttpParams } from '@angular/common/http';
+import { Params } from '@angular/router';
+
+export const getQueryParamStringFromObject = (params: Params): string =>
```

Comment:
Is this needed or should we use the below function directly?

Reply:
It is better to have a separate function, firstly consumers didn't have to write a new instance every time, and secondly if later we want to do standardize queryParams key-value pair we need to change this function only.